drivers/sensor/max17055/max17055.c

```diff
@@ -175,6 +219,7 @@ static int max17055_sample_fetch(const struct device *dev,
 				 enum sensor_channel chan)
 {
 	struct max17055_data *priv = dev->data;
+
```

Comment:
drop blank line

Reply:
Sorry - I'd marked this as resolved because I'd removed it... and `uncrustify` has added it back.

I believe that `uncrustify` is the correct formatter to use?

Reply:
oh well it's not important